CONTRIBUTING.md

```diff
@@ -8,39 +8,46 @@ right place!
 Check out the [README](README.md#usage) for instructions on running the app in
 development.
 
-## Contribution process
-
-### Find something to work on
+## Finding something to work on
 
 Start with issues labelled
 [`good first issue`](https://github.com/jbx-protocol/juice-juicehouse/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
 
-### Create a pull request
+## Getting your pull request reviewed, approved, and merged
 
-To contribute, create a pull request (PR) that targets the `main` branch. A live
-Fleek preview will be automatically deployed for each PR. Before your PR is
-merged, it must meet the following criteria:
+Create a pull request (PR) that targets the `main` branch. A live Fleek preview
+will be automatically deployed for each PR.
 
-- Passes all CI checks.
-- Approved by at least one code owner.
-- Discussed by other design/dev contributors (for contributions that make
-  significant UI/UX changes).
+When your PR has met the [#approval guidelines](approval-guidelines) and is
+ready for review, `@mention` a [codeowner](.github/CODEOWNERS) and ask for a
+review.
 
-### Juicebox app release
+### Git workflow
 
-All changes to the `main` branch will be automatically deployed via
-[Fleek](https://fleek.co).
+We opt for a **rebase policy** where the repository history is kept flat and
+clean. When a feature branch's development is complete, rebase/squash all the
+work down to the minimum number of meaningful commits.
```

Comment:
What do you think of making the default squash and merge. PRs would always enter the main branch as one commit?

Reply:
I don't have a strong opinion either way! If we did do that, we might need to [enable it for the repo](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests).

cc @peripheralist

Reply:
I don't have a _very_ strong opinion, but I _do_ find value in high-quality commit messages over time rather than a limited set of squashed PR titles.

I'm fine either way; typing `rebase` instead of `merge` for upstream changes really doesn't phase me.

Reply:
I prefer keeping feature branches rebased against `main` to keep commit history linear, but for merging _into_ `main` I don't feel too strongly about squashing or not. I often find it helpful to be able to look back through specific commits to see how something worked previously or where it was broken, so I'm leaning towards rebase + no squash.

Reply:
Thanks all. In that case, for this iteration, let's keep this "minimum commit" recommendation in the docs and not alter our merge strategy 👍 . Can always revisit it!